cmd/govim/testdata/scenario_no_completeunimported/no_complete.txt

```diff
@@ -0,0 +1,42 @@
+# Test that completing of unimported std library packages is disabled
+# when CompleteUnimported=0
+
+vim ex 'e main.go'
+
+# Attempt unimported completion
+vim ex 'call cursor(4,1)'
+vim normal Sfmt.Pr
+vim ex 'execute \"normal A\\<C-X>\\<C-O>\\<C-N>\\<C-N>(\\\"Hello\\\")\"'
```

Comment:
As I mentioned on Slack, given we are testing in the context of the `fmt` package which has three functions that share a `Print` prefix and the behaviour of omni-completion, the `<C-n><C-n>` keystrokes are not strictly required. i.e. there are no candidates returned, so firing the keystrokes will be a no-op. 

So perhaps we can remove them in all tests you're adding to remove an potential confusion with the reader thinking we are cycling through results? 

Indeed, might just be worth amending the comment above to something like:

```
# Attempt unimported completion; we should get zero results.
# If we did get results (from fmt) then attempting completion
# would expand from fmt.Pr to fmt.Print, i.e. the longest match
# of the returned candidates
```